components/Poll/PollPreview/PollPreview.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, FlatList } from 'react-native';
+import { Text, Button, Textarea } from 'native-base';
+import { Font } from "expo";
+import styles from "./PollPreviewStyle";
+
+export default class PollPreview extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    let cleanTitle = this.props.data && this.props.data.title ? this.props.data.title : '';
+    let cleanOpts = this.props.data && this.props.data.options ? this.props.data.options : [];
+
+    this.state = {
+      title: cleanTitle,
+      options: cleanOpts,
+    }
+  }
+
+  async componentWillMount() {
+    await Font.loadAsync({
+      Roboto: require("native-base/Fonts/Roboto.ttf"),
+      Roboto_medium: require("native-base/Fonts/Roboto_medium.ttf")
+    });
+  }
+
+  getTotal = (options) => {
+    var total = 0;
+    options.forEach(opt => { total += opt.usersVoted.length; });
+    return total;
+  }
+
+  buildListItems = () => {
+    let items = this.state.options
+            .sort((a, b) => b.usersVoted.length - a.usersVoted.length)
+            .slice(0, 3)
+            .map(item => {
+              if (item._id) {
+                item.key = item._id;
+              }
+              return item;
+            });
+    return items;
+  }
+
+  renderListItem = ({ item }) => {
+    let userIndex = item.usersVoted.findIndex(voter => voter === this.props.loggedInUser._id);
+    var countText;
+    if (userIndex >= 0) {
+      if (item.usersVoted.length > 1) {
+        countText = `You + ${item.usersVoted.length - 1}`
+      } else {
+        countText = 'You'
+      }
+    } else {
+      countText = `${item.usersVoted.length}`
+    }
+    let totalCount = this.getTotal(this.state.options);
+
+    return(
+  		<View style={styles.optionView}>
```

Comment:
Can you fix the formatting/whitespace on this please? :)

Reply:
Should be fixed in most recent update